Taskfile.yml

```diff
@@ -485,7 +485,37 @@ tasks:
     desc: Generate helm manifest using Kustomize for the release
     deps: [controller:generate-kustomize]
     cmds:
-     - "{{.SCRIPTS_ROOT}}/generate-helm-manifest.sh {{.KUBE_RBAC_PROXY}} {{.LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE}} {{.PUBLIC_REGISTRY}} {{.LATEST_VERSION_TAG}} `pwd`/{{.CONTROLLER_ROOT}}"
```

Comment:
I don't think adding a `vars` section is necessary. But it might be useful to add a `preconditions` section that verifies each environment variable is properly defined before allowing the task to proceed.